### PR TITLE
feat(cli): deploy command including org scope

### PR DESCRIPTION
**Pull request by BibiSebi**
## What?
Since past 2 weeks a new feature has been added to the product where organizations are able to create their own field plugins. For this new feature a new API endpoint has been added so we need to consider it inside our CLI to give the user the option to deploy to an organisation. 

## Why?

[JIRA: EXT-1849](https://storyblok.atlassian.net/browse/EXT-1849)

### CODE REVIEW

**Comment by BibiSebi** [comment]
After merging, the version of CLI still needs to be bumped and a new release created

File: packages/cli/src/commands/deploy.ts [hunk 1]
```diff
@@ -291,7 +300,16 @@ const selectApiScope = async (token: string): Promise<Scope> => {
...
+  }).isAuthenticated()
+
+  if (
+    !accessibleToMyPlugins &&
+    !accessibleToPartnerPortal &&
+    !accessibleToOrganizationPortal
+  ) {
     console.error(
       red('[ERROR]'),
       `The token appears to be invalid as it does not have access to either My Plugins or the plugins on the Partner Portal.`,
```
Comment (eunjae-lee) [inline packages/cli/src/commands/deploy.ts:315]: We need to include the organization portal here! :)

Comment (BibiSebi) [reply packages/cli/src/commands/deploy.ts:315]: true :D haha let me quickly add it

**Review by eunjae-lee — APPROVED** [review]
Nice work !

**Pull request merged**